src/main/js/components/header/tableSwitcher/TableSwitcher.jsx

```diff
@@ -89,9 +90,11 @@ class TableSwitcherButton extends React.Component {
     //Show display name with fallback to machine name
     const table = this.props.currentTable;
     const tableDisplayName = table.displayName[this.props.langtag] || (table.displayName[TableauxConstants.FallbackLanguage] || table.name);
-
+    const open_class = (this.state.isOpen) ? "active" : "";
```

Comment:
you could use [`classnames` ](https://www.npmjs.com/package/classnames) here — if you want